content/recipes/introduction/FAIR-and-knowledge-graphs.md

```diff
@@ -249,22 +223,19 @@ alt: wikidata sparql endpoint
 A SPARQL query over Wikidata RDF endpoint: Cell lines with names that could also be URLs (Internet of Cell Lines)
 ```
 ````
-
-For more information about SPARQL queries over Wikidata, see [here](https://www.wikidata.org/wiki/Wikidata:SPARQL_query_service/queries/examples#Cell_lines_with_names_that_could_also_be_URLs_\(Internet_of_Cell_Lines\)).
-     
 ### 2. Property graphs
     
-Like `RDF graphs`, `Property graphs` are too used to build `Knowledge graphs`.
```

Comment:
a small thing is that in the UI, all \`\` are in purple, and hyperlinks are in orange. It's a bit confusing.

Reply:
yes , Martin and Xenia also reported the issue, will be deal with